### PR TITLE
Removes the rescue and construction modules.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Since there's no point in keeping the above two right now, due to general redundancy plus the lack of specialized areas, we're going to be removing them. The remaining modules will remain as is and we'll see if they need to be buffed.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!rescue

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -353,66 +280,6 @@ var/global/list/robot_modules = list(
...
-	src.modules += new /obj/item/gripper/no_use/loader(src)
-	src.modules += new /obj/item/gripper(src)
-	src.modules += new /obj/item/device/t_scanner(src) // to check underfloor wiring
-	src.modules += new /obj/item/device/analyzer(src) // to check air pressure in the area
-	src.modules += new /obj/item/device/lightreplacer(src) // to install lightning in the area
-	src.modules += new /obj/item/device/floor_painter(src)// to make america great again (c)
-	src.modules += new /obj/item/inflatable_dispenser(src) // to stop those pesky humans being entering the zone
-	src.modules += new /obj/item/pickaxe/borgdrill(src) // as station is being located at the rock terrain, which is presumed to be digged out to clear the area for new rooms
-	src.emag = new /obj/item/gun/energy/plasmacutter/mounted(src)
-	src.malfAImodule += new /obj/item/rfd/transformer(src)
```
Comment (SonicGotNuked) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: You're going to remove any possibility for the malf to get the auto borger.

**Comment by Doxxmedearly** [comment]
Consider adding adrenaline to the medication synthesizer for regular medical borg, since they otherwise have no means of restarting a heart. It was added to rescue.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -1001,7 +869,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/inflatable_dispenser(src)
 	// Medical
 	src.modules += new /obj/item/device/healthanalyzer(src)
-	src.modules += new /obj/item/reagent_containers/borghypo/medical(src)
+	src.modules += new /obj/item/reagent_containers/borghypo/rescue(src)
```
Comment (Doxxmedearly) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I don't recommend fixing it like this. Rescue module only contains ("tricordrazine", "norepinephrine", "tramadol", "adrenaline".) That's missing a lot of critical medications in the normal medical borg synth ("bicaridine", "kelotane", "dylovene", "dexalin", "norepinephrine", "tramadol", "deltamivir", "thetamycin")

 I'd go into /reagent_containers/borghydro.dm and just add adrenaline to /borghypo/medical.  Unless it's intentional to drop these meds from the synth, in which case it should probably be changelogged.

**Pull request closed without merging**